test/legacy/roles/scaleway_compute/tasks/main.yml

```diff
@@ -1,6 +1,6 @@
 # SCW_API_KEY='XXX' SCW_ORG='YYY' ansible-playbook ./test/legacy/scaleway.yml --tags test_scaleway_compute
 
 - include_tasks: state.yml
-- include_tasks: ip.yml
-- include_tasks: security_group.yml
-- include_tasks: pagination.yml
+# - include_tasks: ip.yml
```

Comment:
remove comments

Reply:
Testing mistake 😄
Thanks for the heads up